### DISCUSSION TITLE
Kaleido M10 BT 20C Low

### DISCUSSION THREAD

**Discussion by asarule** [artisan-roaster-scope/artisan#1466] [Q&A]
New Kaleido M10 reporting BT temp 20C too low. ET temp likely off too.  Would like to achieve correct readings by means other than symbolic assignment in Config=>Device, Symb ET/BT tab.  When PID roasting would like SV, BT and graph curves to reflect real temps.  Is there a way to "intercept" the signal and make the adjustment?

**Comment by MAKOMO** [comment]
Artisan is logging the values it receives from the machine and I assume that those are what the machine is measuring. How can that be too low!?

Symbolic formulas are exactly the way to "intercept" the signal to make that adjustment.

**Reply by asarule** [reply]
Thanks for your reply.  The reason it "appears to be low" is that FC is reported at approximately 178C on the M10.  Same bean on the Kaffelogic Nano 7 has first crack at 198C - about in the typical FC range.   I have some confusion with symbolic assignment when used with PID roasting and a background created in designer.  For me it would be less confusing if the SV LCD and the symbolic adjusted BT and the background bean curve were displayed as the same value.  I'll try to do a screen capture in the future.

**Reply by hermetheuscoffee** [reply]
Symbolic assignments are indeed what you want to use, even with PID! In your example above, you could go to Config>Device>SymbET/BT and in the BT Y(x) field, enter:
Y2*1.112

This is essentially an 11.2% correction that is applied to every BT temperature reading. Every BT temperature reading will get multiplied by 1.112 and the resulting value is what the PID will use to match to the background SV. Since you hit FC at 178 degrees, you can see that 178 x 1.112 = 197.9, which is the target temp you want!

**Comment by Terracotta-6** [comment]
BT temperature is relative temperature, temperature  linear compensation regulation, but not a good method.
Calibrate by adjusting the depth of the BT thermocouple inserted into the boiler.
Contact the after-sales service to obtain the method of adjusting BT temperature.

**Reply by Terracotta-6** [reply]
https://www.kaleidosniper.com/
